care/facility/models/daily_round.py

```diff
@@ -418,6 +439,15 @@ class InsulinIntakeFrequencyType(enum.Enum):
         default=list, validators=[JSONFieldSchemaValidator(NURSING_PROCEDURE)]
     )
 
+    length = models.FloatField(default=0.0, validators=[MinValueValidator(0)])
```

Comment:
As per https://github.com/coronasafe/care_fe/issues/3432, each section must have these values, so we might have to dump these as a json. storing these as columns only allows us to store one instance of the value.

Reply:
@aparnacoronasafe Discuss with @iamsdas to understand exactly what needs to be stored and how it needs to be represented.  ( I might be wrong and this might be correct )

Reply:
@iamsdas @vigneshhari Yes, there will be multiple windows with these parameters, one for each pressure sore. Added clarification on the issue